### PR TITLE
Reusable Button and Header

### PR DESCRIPTION
**Pull request by youssefayman22**
- Made a reusable header that recieves the page title and can navigate backwards

image

- Added the reusable Button component
image

### CODE REVIEW

File: coffee-shop/src/genericComponents/Header/Header.jsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import React from "react";
```
Comment (arahman10783) [inline coffee-shop/src/genericComponents/Header/Header.jsx:1]: you need to add index file in the header component folder as we did in pages

**Review by arahman10783 — CHANGES_REQUESTED** [review]

File: coffee-shop/src/genericComponents/Button/Button.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import styles from "../Button/Button.module.css";
+
+const Button = (props) => {
```
Comment (arahman10783) [inline coffee-shop/src/genericComponents/Button/Button.js]: I think you will need to pass a function as a props to use onClick, and also you can use destructuring for the props 
`({ label, handleClick}) => { .....`

File: coffee-shop/src/genericComponents/Header/Header.jsx (hunk 1 above)
Comment (youssefayman22) [reply coffee-shop/src/genericComponents/Header/Header.jsx:1]: done

File: coffee-shop/src/pages/Order/Order.jsx [hunk 3]
```diff
@@ -1,12 +1,14 @@
 import React from "react";
 import CustomizedRadioButton from "./components/CustomizedRadioButton/CustomizedRadioButton";
+import Button from "../../genericComponents/Button/Button";
```
Comment (arahman10783) [inline coffee-shop/src/pages/Order/Order.jsx]: as you added index in button component you just import it from the index like that 
`import Button from "../../genericComponents/Button/";`

**Review by arahman10783 — CHANGES_REQUESTED** [review]

File: coffee-shop/src/genericComponents/Button/Button.js [hunk 4]
```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import styles from "../Button/Button.module.css";
+
+const Button = ({ label, handleClick }) => {
+  return (
+    <div className={styles.container}>
+      <button onClick={handleClick} className={styles.button}>
+        {label.text}
```
Comment (arahman10783) [inline coffee-shop/src/genericComponents/Button/Button.js]: you need to use text or label, and I recommend label, `label.text` will return error

File: coffee-shop/src/pages/Order/Order.jsx [hunk 5]
```diff
@@ -1,12 +1,14 @@
...
 
 const Order = () => {
   return (
-    <div>
-      Order
+    <>
+      Order page
       <CustomizedRadioButton />
-    </div>
+      <Button text="Order" />
```
Comment (arahman10783) [inline coffee-shop/src/pages/Order/Order.jsx]: if you used label you need to add the props here as label not text

**Review by arahman10783 — CHANGES_REQUESTED** [review]

**Review by arahman10783 — APPROVED** [review]

**Pull request merged**